plugin.video.regiotv/resources/settings.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<settings>
+
+  <category label="30860"> <!--Integration -->
+    <setting label="30861" type="lsep"/> <!-- Integration with other add-ons -->
+    <!-- IPTV Manager -->
+    <!-- setting label="30875" help="30876" type="action" action="InstallAddon(service.iptv.manager)" option="close" visible="!System.HasAddon(service.iptv.manager)"/ --> <!-- Install IPTV Manager add-on -->
+    <setting label="30877" help="30878" type="bool" id="iptv.enabled" default="true" visible="String.StartsWith(System.BuildVersion,18) + System.HasAddon(service.iptv.manager) | System.AddonIsEnabled(service.iptv.manager)" />
```

Comment:
Where are all these settings used? I cannot seem to find it?

Reply:
It is used by IPTV Manager to determine the add-on is supported.